client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -75,7 +99,19 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             onChange={handleChange}
           />
 
-          <Box textAlign="center" marginTop={5}>
+          <Box className={classes.buttonsWrapper} marginTop={5}>
+            <Button
+              onClick={() => {
+                updateLoginContext(hardCodedData.success);
```

Comment:
Rather than updating the login context with mock data, let's create a user in the db and hardcode those credentials into the `login` helper - which will be called when we click on `Demo`